### PR TITLE
[fuzz] merge both asn1 fuzzers into one

### PR DESCRIPTION
**Pull request by deweerdt**

### CODE REVIEW

**Comment by deweerdt** [comment]
@jfoote I've added coverage to certificate loading, this should be large enough

File: fuzz/fuzz-asn1.c [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+{
+    int *c = ctx;
+    va_list argptr;
+    va_start(argptr, format);
+    c += vsnprintf(NULL, 0, format, argptr);
+    va_end(argptr);
+    return;
+}
+
+int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size);
```
Comment (jfoote) [inline fuzz/fuzz-asn1.c]: Is this declaration necessary?

Comment (jfoote) [reply fuzz/fuzz-asn1.c]: Update: I confirmed it is not. I saw I had the ability to push to the branch so I just removed it.

**Comment by jfoote** [comment]
This has been integrated into oss-fuzz https://github.com/google/oss-fuzz/pull/2236, thanks @deweerdt!

**Pull request merged**